config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -88,3 +88,20 @@ presubmits:
         - "--scenario=execute"
         - "--"
         - "./scripts/ci-is-vendor-in-sync.sh"
+  - name: pull-cluster-api-integration
```

Comment:
Since we're introducing a new job, we should probably make it optional to start. We can always follow up later to make it a required job after verifying it doesn't cause issues/flake.

Reply:
@detiber yes, totally agree, `always_run` is `false` currently, so it will not run until triggered by `/test pull-cluster-api-integration` at PR.

Reply:
Ah, I was expecting to see `optional: true`. Setting it to run by default would also give us better signal to how the test is performing.

Reply:
got it, thanks